Feature/2326/add new hotkey to save in clipboard (#2353)
MaibornWolff/codecharta: CHANGELOG.md

### Added 🚀

-   A changelog dialog with the latest additions to CodeCharta appears on version update ([#1315](https://github.com/MaibornWolff/codecharta/issues/1315))

> -   An option has been added to the global configuration to enable coping screenshot to clipboard instead of saving it in a file ([#2326](https://github.com/MaibornWolff/codecharta/issues/2326))

Reviewer: ```suggestion
-   An option has been added to the global settings to enable copying screenshots to clipboard instead of saving them in a file ([#2326](https://github.com/MaibornWolff/codecharta/issues/2326))
```
